### DISCUSSION TITLE
how to disable registration

### DISCUSSION THREAD

**Discussion by mckaygerhard** [filebrowser/filebrowser#1581] [Q&A] [answered]
i noted that users can register own account, this can be disabled without manage recompilation? i means  without modify the sources!

**Comment by ramiresviana** [comment]
The signup option is available on the Settings page under the Global Settings section, it can also be changed using the Command Line Interface. When enabled, users can create new accounts using the signup form available on the login page.

**Reply by mckaygerhard** [reply]
i cannot see that opcion.. over gui.. neither in command line.. also i try some "flags" and it seems the db does got corupted when are running..

also i guess the documentation web page is outdated.. so i ask here exact command line by example

`filebroser c /etc/share/config.json` give me error

**Reply by mckaygerhard** [reply]
ok got it the "allow users to sing up" but at the command line doe snot work.. what is the exact line? i do not find on https://filebrowser.org/cli/filebrowser-config-set

**Reply by ramiresviana** [reply]
The `Allow users to signup` is the first option on the Global Settings section of the web interface. The command `filebrowser config set --signup=true` can be used to disable it, keep in mind that CLI cannot be used when FileBrowser is running.

**Reply by mckaygerhard** [reply]
> --signup=true

puff the documentation and your references are wrong but found the solution after WASTE OF TIME:

this from command line hel `  -s, --signup                             allow users to signup` is confused, and this `filebrowser config set --signup=true` give me a hint (cos was wrong but THANKS @ramiresviana  provide me right directions)

**Comment by mckaygerhard** [comment, marked as answer]
SOLUTION:` filebrowser config set --signup=false` for disable the registration.  https://github.com/filebrowser/filebrowser/discussions/1581#discussioncomment-1442445

> --signup=true

puff the documentation and your references are wrong but found the solution after WASTE OF TIME:

this from command line hel `  -s, --signup                             allow users to signup` is confused, and this `filebrowser config set --signup=true` give me a hint (cos was wrong but THANKS @ramiresviana  provide me right directions)
